README.md

```diff
@@ -88,6 +87,55 @@ Then, set up the environment variables and run the test suite:
 $ sbt test
 ```
 
+Publishing
+----------
+
+This application uses Docker for deployment. To create a Docker image, use the
+following command:
+
+```console
+$ docker build -t codingteam/loglist:$LOGLIST_VERSION -t codingteam/loglist:latest -f loglist.dockerfile .
+```
+
+(where `$LOGLIST_VERSION` is the version for the image to publish)
+
+Then push the image to the Docker Hub:
+
+```console
+$ docker login # if necessary
+$ docker push codingteam/loglist:$LOGLIST_VERSION
+$ docker push codingteam/loglist:latest
+```
+
+Deployment
+----------
+
+To install the application from Docker, run the following command:
+
+```console
+$ docker run -d --restart unless-stopped --name $NAME -v $CONFIG:/app/conf/application.conf --env-file $ENV_FILE --net host codingteam/loglist:$VERSION
```

Comment:
`--net host` is pretty powerful, why not just `--publish 9000`?

Reply:
We need `--net host` to acces PostgreSQL instance (which isn't published to the container on our production site).

Any ideas how to solve it without `--net host`? I've read about some magic tricks, but the best of them seem to only work on Docker 20.04 (while we have pretty old version in production).

Reply:
It's a bit of a hassle, but we could crate a separate bridge just for this container, and re-configure our PostgreSQL instance to be accessible from that bridge as well. I tried to quickly find an example of this, but apparently people just bind Postgre to bridge0 which is shared by *all* the containers.

Reply:
Thanks, I'll take a look.

Reply:
Fixed.